syntax = "proto3";

package website_content.v1;

import "common.proto";

option go_package = "example.com/website_content/v1;website_content_v1";

message PortfolioItem {
  string id = 1;
  Image image = 2;
  TitledBlock details = 3;
}
